packages/arb-node-core/ethbridge/auth.go

```diff
@@ -92,7 +171,7 @@ func (t *TransactAuth) makeContract(ctx context.Context, contractFunc func(auth
 
 		t.auth.Nonce = t.auth.Nonce.Add(t.auth.Nonce, big.NewInt(1))
 		auth.Nonce = t.auth.Nonce
-		addr, tx, _, err = contractFunc(auth)
+		err = t.sendTx(ctx, tx)
```

Comment:
Is this going to actually effect the nonce change? We might just want to get rid of this code IMO, I've never seen it actually used.

Reply:
We ran into nonce issues early on that were mostly tracked down to running multiple validators with same wallet.  I agree that it is good to remove this code, because nonce issues are usually a symptom of something else being wrong, so better to not gloss over it.